ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/components/ScheduleDetails.tsx

```diff
@@ -37,7 +37,7 @@ function ScheduleDetails({ formValues }: ScheduleDetailsProps): ReactElement {
     return (
         <Flex direction={{ default: 'column' }}>
             <FlexItem>
-                <Title headingLevel="h3">Schedule details</Title>
+                <Title headingLevel="h2">Schedule details</Title>
```

Comment:
Ditto `h3` versus `h2`

Reply:
Fixed. Details sections will use h2 and the wizard review step will use h3